### ISSUE TITLE
JDK10 and proxy-vole

### ISSUE THREAD

**Issue by mejmo** [MarkusBernhardt/proxy-vole#51]
As proxy-vole has a dependency `delight-nashorn-sandbox` which is using `com.sun.tools 1.4.2`, maven compilation ends with `Could not find artifact com.sun:tools:jar:1.4.2 at specified path`. Is there any fix except for adding a dependency with system scope? (since it is in generla bad idea always).

**Comment by gschnepp** [comment]
Yes. Just define another, newer dependency for delight-nashorn-sandbox aside proxy-vole. I had the same problem and Max Rohde has created a newer version without this dependency for me. Works for me with .15 or .16, I don't remember exactly.
